docs/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale.md

```diff
@@ -59,6 +58,12 @@ You can reduce the number of `RoleBindings` in the upstream cluster in the follo
 * Kubernetes permissions are always "additive" (allow-list) rather than "subtractive" (deny-list). Try to minimize configurations that gives access to all but one aspect of a cluster, project, or namespace, as that will result in the creation of a high number of `RoleBinding` objects.
 * Experiment to see if creating new projects or clusters manifests in fewer `RoleBindings` for your specific use case.
 
+#### Use external authentication
+
+For any use case with more than 50 users configuring an [external authentication](../../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/authentication-config.md) provider with support of groups is necessary to attain good performance.
+
+After configuring external authentication, make sure permissions are assigned to groups instead of individual users, as that will reduce the count of `RoleBinding` objects that Rancher will create.
```

Comment:
```suggestion
After you configure external authentication, make sure to assign permissions to groups instead of to individual users.  This helps reduce the `RoleBinding` object count.
```